src/Network/Request.php

```diff
@@ -283,12 +285,20 @@ protected function _processPost($data)
         }
         if ($this->env('HTTP_X_HTTP_METHOD_OVERRIDE')) {
             $data['_method'] = $this->env('HTTP_X_HTTP_METHOD_OVERRIDE');
+            $override = true;
         }
         $this->_environment['ORIGINAL_REQUEST_METHOD'] = $method;
         if (isset($data['_method'])) {
             $this->_environment['REQUEST_METHOD'] = $data['_method'];
             unset($data['_method']);
+            $override = true;
+        }
+
+        $method = $this->_environment['REQUEST_METHOD'];
+        if ($override && !in_array($method, ['PUT', 'POST', 'DELETE', 'PATCH'])) {
+            $data = [];
```

Comment:
What about input decoded from `php://input`? That will still have the same issue.